spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'ilham', photo: 'www.pp.com', bio: 'student from Morocco') }
+
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'postCounter must be integer greater than or equal zero' do
+    subject.postsCounter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'postCounter must be integer greater than or equal zero' do
+    subject.postsCounter = 0
+    expect(subject).to be_valid
+  end
+end
```

Comment:
- [ ] Kindly add a test for return_recent_posts method. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)